frontend/test/metabase/visualizations/components/settings/ChartNestedSettingsSeries.unit.spec.js

```diff
@@ -17,28 +18,28 @@ function getSeries(display) {
 }
 describe("ChartNestedSettingSeries", () => {
   it("shouldn't show line/area/bar buttons for row charts", () => {
-    const settings = mount(
+    const { queryByRole } = render(
       <ChartSettings
         series={getSeries("row")}
         initial={{ section: "Display" }}
       />,
     );
 
-    expect(settings.find(".Icon-line")).toHaveLength(0);
-    expect(settings.find(".Icon-area")).toHaveLength(0);
-    expect(settings.find(".Icon-bar")).toHaveLength(0);
+    expect(queryByRole("img", { name: /line/i })).not.toBeInTheDocument();
+    expect(queryByRole("img", { name: /area/i })).not.toBeInTheDocument();
+    expect(queryByRole("img", { name: /bar/i })).not.toBeInTheDocument();
   });
 
   it("should show line/area/bar buttons for bar charts", () => {
-    const settings = mount(
+    const { getByRole } = render(
       <ChartSettings
         series={getSeries("bar")}
         initial={{ section: "Display" }}
       />,
     );
 
-    expect(settings.find(".Icon-line")).toHaveLength(1);
-    expect(settings.find(".Icon-area")).toHaveLength(1);
-    expect(settings.find(".Icon-bar")).toHaveLength(1);
+    getByRole("img", { name: /line/i });
```

Comment:
Probably requires a RTL plugin, but it could be nice to have `*ByIconName("line")`

Reply:
We can probably make a helper function for that.